src/services/orderItemService.ts

```diff
@@ -1,11 +1,42 @@
 import { findOrderItemsByOrderId } from "../reposetories/orderItemRepository";
+import Order from "../models/Order"; // Assuming `Order` is your order model
 
 export const getOrderItemsService = async (orderId: string) => {
-  const orderItems = await findOrderItemsByOrderId(orderId);
-
-  if (!orderItems.length) {
-    throw new Error("No order items found for the given order ID");
-  }
-
-  return orderItems;
-};
+    const orderItems = await findOrderItemsByOrderId(orderId);
```

Comment:
Validate the order ID before starting the operations

Reply:
ok i will but user cannot place order without adding at least one order item